lxd/db/containers.go

```diff
@@ -1237,15 +1238,15 @@ func (c *Cluster) ContainerBackupCreate(args InstanceBackupArgs) error {
 			optimizedStorageInt = 1
 		}
 
-		str := fmt.Sprintf("INSERT INTO instances_backups (instance_id, name, creation_date, expiry_date, container_only, optimized_storage) VALUES (?, ?, ?, ?, ?, ?)")
+		str := fmt.Sprintf("INSERT INTO instances_backups (instance_id, name, creation_date, expiry_date, container_only, optimized_storage, backup_compression_algorithm) VALUES (?, ?, ?, ?, ?, ?, ?)")
 		stmt, err := tx.tx.Prepare(str)
 		if err != nil {
 			return err
 		}
 		defer stmt.Close()
 		result, err := stmt.Exec(args.InstanceID, args.Name,
 			args.CreationDate.Unix(), args.ExpiryDate.Unix(), instanceOnlyInt,
-			optimizedStorageInt)
+			optimizedStorageInt, args.CompressionAlgorithm)
```

Comment:
Same here, that bit shouldn't be needed at all.

Reply:
went in by mistake, removed